src/std/env.ab

```diff
@@ -76,52 +89,59 @@ pub fun is_root(): Bool {
     return false
 }
 
+/// ?
```

Comment:
printf c command with format in first parameters and array of string
https://www.man7.org/linux/man-pages/man1/printf.1.html